### DISCUSSION TITLE
Not able to get response after login to PostgreSQL Cluster Console

### DISCUSSION THREAD

**Discussion by gmasand** [autobase-tech/autobase#781] [General]
I have created a ec2 instance in AWS cloud and started docker container using below command

docker run -d --name pg-console \
  --publish 80:80 \
  --publish 8080:8080 \
  --env PG_CONSOLE_API_URL=http://localhost:8080/api/v1 \
  --env PG_CONSOLE_AUTHORIZATION_TOKEN=psqlsecret \
  --env PG_CONSOLE_DOCKER_IMAGE=vitabaks/postgresql_cluster:latest \
  --volume console_postgres:/var/lib/postgresql \
  --volume /var/run/docker.sock:/var/run/docker.sock \
  --volume /tmp/ansible:/tmp/ansible \
  --restart=unless-stopped \
  vitabaks/postgresql_cluster_console:2.0.0

I am trying to open the PosgreSQL Console from my we browser

http://<public IP of ec2 instance>

I am able to enter inside the console but as soon as I click on any of the buttons to create a cluster I am not getting any response.

Can someone help me here ?

Regards
Garima

**Comment by vitabaks** [comment]
In this case, replace localhost with your server's IP address in the `PG_CONSOLE_API_URL` variable.

**Reply by gmasand** [reply]
It worked thanks for quick help

**Comment by gmasand** [comment]
Problem resolved
